lacommunaute/forum_search/tests/test_search_indexes.py

```diff
@@ -202,3 +208,9 @@ def test_unapproved_post_is_exclude(client, db, search_url):
     call_command("rebuild_index", noinput=True, interactive=False)
     response = client.get(search_url, {"q": "emplois"})
     assertContains(response, "Aucun résultat")
+
+
+def test_extra_context(client, db, search_url):
+    ForumFactory()
+    response = client.get(search_url)
+    assertContains(response, "forum")
```

Comment:
On pourrait ajouter un peu de context pour être sûr de l’emplacement de forum. Sur un forum, le mot peut apparaître à plein d’endroit.
Est-ce que tu souhaitais vérifier la présence de :
`<label for="id_m_2"><input type="radio" name="m" value="forum" id="id_m_2">
 la documentation</label>` ?

Reply:
ok avec ajouter du contexte. Le forum créé est necessaire pour le nouveau bouton "poser une question" (`forum.slug` et `forum.id` dans le retournement de l'url). J'ajoute.